CompetitionTemplate/src/competition/subsystems/collector/commands/CollectorIntakeCommand.java

```diff
@@ -29,7 +32,7 @@ public void end() {
     @Override
     public void execute() {
         if (enableCollectorAutoStop.get()) {
-            if (!collectorSubsystem.isBallInCollector()) {
+            if (this.timeSinceInitialized() < autoStopOverrideTime.get() || !collectorSubsystem.isBallInCollector()) {
```

Comment:
The goal here is twofold:
- Wedge the ball into the mechanism when it is acquired -- the sensor _does_ trigger too early.
- Allow the driver to pull the ball back in then they are going over a defense. Just releasing and clicking it a few times while bouncing around has been demonstrated to make it much more difficult to bump free.